lib/streamlit/components/types/base_custom_component.py

```diff
@@ -102,6 +112,7 @@ def create_instance(
         *args,
         default: Any = None,
         key: str | None = None,
+        on_change_handler: WidgetCallback | None,
```

Comment:
I think we probably want to call this `on_change` and not `on_change_handler` to better align with APIs of other commands.

Reply:
@sfc-gh-jcarroll @jrieke Lukas' point makes sense to me. Could you have a quick look and confirm this, please?